### ISSUE TITLE
Edit Wallet Name & Scan my QR Code on Menu does not have Accessibility Label or TestID

### ISSUE THREAD

**Issue by nodlesh** [bcgov/bc-wallet-mobile#1510]
### Description of problem
The Edit Wallet Name pencil icon and "Scan my QR Code" needs a testID and an Accessibility label

### Screenshots and/or log output
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBZy95IiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--ef05a89e62efc03c8bda5fc9baf58c588d6be951/Screenshot%202023-09-29%20at%205.28.34%E2%80%AFPM.png)

### Environment
- [ ] Occurs on Android
- [ ] Occurs on iOS

Build #:

Android Device Model: 

iOS Device Model: 

### Workaround
Using a text based locator in the tests for the edit button but that is not a long term solution.

### Severity
- [ ] High
- [x] Medium
- [ ] Low

**Comment by nodlesh** [comment]
@jleach The Edit Wallet Name pencil on the Scan my QR Code page also needs a TestID. Do you want to do that in this ticket or a new one? 
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNmNGQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--4c2054c56b3cf2484836f68e7f538fd36ed3d340/Screenshot%202023-10-04%20at%201.38.00%E2%80%AFPM.png)

**Comment by jleach** [comment]
@nodlesh My Appium Inspector no longer works so I had to wing this one. LMK if the TestIDs don't appear for you.

**Comment by nodlesh** [comment]
@jleach I have a tested on the menu page for the edit wallet name pencil, however I do not have one for the edit pencil on the "My QR Code" page as I posted in the above comment. Was that an oversight or was it fixed and I'm not seeing it?
